viewing_party/party.py

```diff
@@ -1,23 +1,161 @@
 # ------------- WAVE 1 --------------------
 
+from re import U
+from venv import create
+
+
 def create_movie(title, genre, rating):
-    pass
+    movie = {
+        "title" : title,
+        "genre" : genre,
+        "rating" : rating,
+    }
+    if all(movie.values()) == False:
+        return None
+    else:
+        return movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
 
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, movie):
+    for movie_dict in user_data["watchlist"]:
+        if movie_dict["title"] == movie:
+            user_data["watched"].append(user_data["watchlist"].pop())
+    return user_data
+

```

Comment:
There's a bit a of a bug here: `pop` always removes the last element from a list, but it might be the case that the movie we're looking to remove is somewhere in the middle. In that case, it could remove the wrong movie.

Also, we generally want to avoid modifying a list as we're iterating over it using a `for` loop. It can cause some really hard to debug issues! In general, it's better to identify the items that we want to remove, then actually perform the removals outside of the loop.